src/HomeScreen/hooks/useAppFirstTime.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const useAppFirstTime = () => {
+  const [isFirstTime, setIsFirstTime] = useState(true);
+  useEffect(() => {
+    (async () => {
+      const value = await AsyncStorage.getItem("hasOpenedAppBefore");
+      if (value !== null) {
+        setIsFirstTime(false);
+      } else {
+        setIsFirstTime(true);
```

Comment:
I think, you can use ternary operator to avoid multi line if condition like 
`value == null ? setIsFirstTime(true) : setIsFirstTIme(false)`

Reply:
fixed